crates/flipperzero/src/notification/messages.rs

```diff
@@ -0,0 +1,61 @@
+use super::{Light, NotificationMessage};
```

Comment:
The choice of re-exporting the results of the `NotificationMessage` constructors with no inputs was mainly made to keep parity with the c API. However, I believe that this also increases the user experience somewhat as it enables the user to use a single export for all default messages if they only need and/or want those but I do see your point. What do you think of making the `const`s that depend on parameters associated constants and scrapping the ones with no parameter to maybe get the best of both worlds?

Reply:
If there is a desire to keep or provide parity with the C API, that might be better served with a prelude-style module designed for `use flipperzero::notification::c_prelude::*;` usage, that collects / renames constants from other parts of the API. This would be a fine place for the constants that are duplicative of `const fn`s to live.

As far as designing a "native" Rust API, grouping based on semantics rather than structure feels like it gives a more intuitive API (like my suggestion further down about module renaming). In an ideal world, the user wouldn't need to care much about whether a provided default corresponds to a single `NotificationMessage` or multiple; they could just compose the messages as they need. We don't live in an ideal world, but macros let us get close.

> What do you think of making the consts that depend on parameters associated constants

Associated constants of which types?

Reply:
Good point but as discussed with `play_sound!` splatting sequences into each other is very difficult, especially doing so for arbitrary sequences due to slice length etc. therefore I would've preferred to keep messages and sequences separate but since `notification_sequence!` is type checked I guess it would not be that much of an issue.

Reply:
I agree that for the `flipperzero` crate, there is no need for 1:1 party with the C API (some things might be unsafe or have a more idiomatic way to do in Rust). But the intent is to expose as much of features as (safely) possible.

It might even make sense to eventually have an alternate API that's a little more friendly than this one (e.g. `Display::set_backlight`). `NotificationMessage` lists aren't exactly the nicest things to work with.